### PR TITLE
Setup  sass-based css design system #49

### PR DESCRIPTION
**Pull request by nancymuyeh**
@stephane-segning , PR for the implementation of sass-based css design system.

### CODE REVIEW

**Review by stephane-segning — CHANGES_REQUESTED** [review]
Please remove everything about gulp, as react vite already supports sass

File: docs/sass_usage.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+- [node.js] (https://nodejs.org/en/):javascript runtime enviroment, the foundation on which sass is built.
+
+- [npm] (https://www.npmjs.com/): Package manager for node.js
+
+- [gulp] (https://gulpjs.com/docs/en/getting-started/quick-start/):Javasript toolkit that helps automate the running of task.It can be useed to watch a .scss file and compile it to a .css file automatically.  **note** gulp is optional here as other tools can be used for example visualsstudio code has an extention for sass. sass itself has commands that can be run on the terminal to compile from .scss to .css.
+
+- [sass] (https://sass-lang.com/install/) :The preprocessor programming language that is either interpreted or compiled into css.
+
+# installation
```
Comment (stephane-segning) [inline docs/sass_usage.md:19]: You should probably also update your doc, because this is very general. In our case, we're using `vite` & `react`

**Review by stephane-segning — CHANGES_REQUESTED** [review]

File: power-pay-frontend/sass/index.scss
[the remark this answers is not in the document]
Comment (nancymuyeh) [reply power-pay-frontend/sass/index.scss:1]: Should I remove the index.scss file as well?

**Review by stephane-segning — CHANGES_REQUESTED** [review]
Good work overall. Better doc too!

File: power-pay-frontend/src/index.scss
[the remark this answers is not in the document]
Comment (nancymuyeh) [reply power-pay-frontend/src/index.scss:1]: Done

**Review by stephane-segning — APPROVED** [review]
Awesome! Thank you @nancymuyeh!

**Pull request merged**